### PR TITLE
Footer nav changes on Core

### PR DESCRIPTION
**Pull request by holder-josh**
**Before:**
before

**After:**
after

cc @stephanfowler

### CODE REVIEW

File: static/src/stylesheets/module/nav/_navigation.scss [hunk 1]
```diff
@@ -737,6 +737,11 @@ $c-navigation-expandable-background: colour(neutral-1);
             min-width: $navigation-toggler-width;
         }
     }
+
+    .is-not-modern .l-footer & {
+        min-width: $navigation-toggler-width;
+        font-size: 24px;
```
Comment (OliverJAsh) [inline static/src/stylesheets/module/nav/_navigation.scss]: Should we use the `fs` mixins? https://github.com/guardian/guss-typography

**Comment by OliverJAsh** [comment]
:+1: Nice

File: static/src/stylesheets/module/nav/_navigation.scss (hunk 1 above)
Comment (Jholder112233) [inline static/src/stylesheets/module/nav/_navigation.scss]: I would usually but this is a magic number to get the icon to line up correctly.

Comment (OliverJAsh) [inline static/src/stylesheets/module/nav/_navigation.scss]: In that case could you leave a comment to explain why you have this?

On Fri, 2 Oct 2015 at 17:00 Josh Holder <notifications@github.com> wrote:

> In static/src/stylesheets/module/nav/_navigation.scss
> <https://github.com/guardian/frontend/pull/10757#discussion_r41037350>:
>
> > @@ -737,6 +737,11 @@ $c-navigation-expandable-background: colour(neutral-1);
> >              min-width: $navigation-toggler-width;
> >          }
> >      }
> > +
> > +    .is-not-modern .l-footer & {
> > +        min-width: $navigation-toggler-width;
> > +        font-size: 24px;
>
> I would usually but this is a magic number to get the icon to line up
> correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/10757/files#r41037350>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

Comment (Jholder112233) [inline static/src/stylesheets/module/nav/_navigation.scss]: Done.

**Pull request merged**